costmap:
  global_frame: map
  robot_base_frame: base_footprint
  update_frequency: 20.1
  publish_frequency: 10.1
  static_map: false
  rolling_window: true
  width: 6.0
  height: 6.0
  resolution: 0.05
  transform_tolerance: 0.5
  
  plugins:
    - {name: obstacle_layer, type: "custom_costmap::ObstacleLayer"}
    - {name: dynamic_inflation_layer, type: "costmap_2d::InflationLayer"}
